src/main/java/Echo.java

```diff
@@ -0,0 +1,175 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
+public class Echo {
+    static String break_line = "----------------------------------------";
```

Comment:
You could consider capitalizing the naming of string literals to follow this [guideline](https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-magic-numbers)